packages/application-components/src/components/dialogs/internals/dialog.styles.ts

```diff
@@ -2,10 +2,34 @@ import { css } from '@emotion/react';
 import { customProperties } from '@commercetools-uikit/design-system';
 
 type StyleProps = {
-  size: 'm' | 'l' | 'scale';
+  size: 'm' | 'l' | 7 | 8 | 9 | 10 | 'scale';
+};
+
+const getConstraintForGridStyle = (size: StyleProps['size']) => {
+  switch (size) {
+    case 7 || 'm':
+      return customProperties.constraint7;
+    case 8:
+      return customProperties.constraint8;
+    case 9:
+      return customProperties.constraint9;
+    case 10 || 'l':
+      return customProperties.constraint10;
+    case 'scale':
+      return customProperties.constraintScale;
+    default:
+      return customProperties.constraintScale;
+  }
 };
 
 export const getModalContentStyles = (props: StyleProps) => {
+  const sizeConstraint = getConstraintForGridStyle(props.size)
+  const gridSize = props.size === 'scale'
+      ? css`[row3-end] / ${customProperties.spacingXl} 1fr ${customProperties.spacingXl}`
```

Comment:
Hmm, I think the css function might not work here like this, because this is only a segment of a css rule (of `grid`)